docs/roman/references_general/distortion_reffile.rst

```diff
@@ -0,0 +1,45 @@
+:orphan:
+
+.. _distortion_reffile:
+
+DISTORTION Reference File
+-------------------------
+
+:REFTYPE: DISTORTION
+:Data model: `~roman_datamodels.datamodels.DistortionRefModel`
+
+The distortion reference file contains a combination of astropy models,
+representing the transform from detector to the telescope V2, V3 system.
+The following convention was adopted:
+
+- The output in the V2, V3 system is in units of arcsec.
```

Comment:
Remove this line, since we don't know the units yet

Reply:
Maybe just?
The output is in the V2, V3 system.
and we can add units later.